kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -160,18 +160,21 @@
         });
       },
       setSuggestionTerm(newVal, oldVal) {
-        if (newVal.length < 3) {
-          // Don't search for suggestions if less than 3 characters entered
-          this.suggestionTerm = '';
-          this.usernameSuggestions = [];
-        } else if ((!newVal.startsWith(this.suggestionTerm) && this.suggestionTerm.length) ||
-          !this.suggestionTerm.length) {
-          // We have already set a suggestion search term
-          // The currently set suggestion term does not match the current username
-          // Or we do not currently have a suggestion term set
-          // Set it to the new term and fetch new suggestions
-          this.suggestionTerm = newVal;
-          this.setSuggestions();
+        if (newVal) {
```

Comment:
not familiar with the code, does this need an `else` clause to clear suggestions if they clear the username?